cli/sign.go

```diff
@@ -96,6 +107,27 @@ func Sign(operators []string, depositData []byte, log shared.QuietLogger) ([]api
 	}
 }
 
+// parseOperator takes a string in form `$validatorNonce,$address` and separates it out
+// e.g. "4,https://example.com" returns `4, https://example.com, nil`
+func parseOperator(input string) (uint32, string, error) {
+	parts := strings.Split(input, ",")
+	l := len(parts)
+
+	if l < 2 {
+		return 0, "", errors.New("operator tuple didn't have enough commas in it")
+	}
+	if l > 2 {
+		return 0, "", errors.New("operator tuple had too many commas in it")
+	}
+
+	validatorNonce, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return 0, "", errors.New("validatorNonce for the operator must be a number")
```

Comment:
any reason not to wrap the existing `err`?
```suggestion
		return 0, "", fmt.Errorf("validatorNonce for the operator must be a number: %w", err)
```

Reply:
it bubbles through to the CLI, so I didn't wanna expose anything to mad to the user